src/services/util/helpers.tsx

```diff
@@ -35,9 +42,23 @@ export function formatBalance(
     : 0;
 }
 
+/**
+ * @deprecated - switch to formatUnitsWithDecimals
```

Comment:
I understand the reason that's has been done, but should we migrate things in scope of this PR?
I mean switch to `parseUnitsWithDecimals`.

Reply:
I took a gradual approach by introducing the new method and marking old one as deprecated, so we can cover the codebase in some steps, without blowing up the scope of current PR too much